arm-eventhub/2017-04-01/swagger/EventHub.json

```diff
@@ -0,0 +1,1930 @@
+              "$ref": "#/definitions/RegenerateAccessKeyParameters"
+            },
+            "description": "Parameters required to regenerate the connection string."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Connection strings regenerated successfully.",
+            "schema": {
+              "$ref": "#/definitions/AccessKeys"
+            }
+          },
+          "default": {
+            "description": "Eventhub error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.EventHub/namespaces/{namespaceName}/eventhubs": {
+      "get": {
+        "tags": [
+          "EventHubs"
+        ],
+        "operationId": "EventHubs_ListByNamespace",
+        "x-ms-examples": {
+          "EventHubsListAll": { "$ref": "../examples/EventHubs/EHEventHubListByNameSpace.json" }
+        },
+        "description": "Gets all the Event Hubs in a Namespace.",
+        "externalDocs": {
+          "url": "https://msdn.microsoft.com/en-us/library/azure/mt639493.aspx"
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/namespaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the list of Event Hubs.",
+            "schema": {
+              "$ref": "#/definitions/EventHubListResult"
+            }
+          },
+          "default": {
+            "description": "Eventhub error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.EventHub/namespaces/{namespaceName}/eventhubs/{eventHubName}": {
+      "put": {
+        "tags": [
+          "EventHubs"
+        ],
+        "operationId": "EventHubs_CreateOrUpdate",
+        "x-ms-examples": {
+          "EventHubCreate": { "$ref": "../examples/EventHubs/EHEventHubCreate.json" }
+        },
+        "description": "Creates or updates a new Event Hub as a nested resource within a Namespace.",
+        "externalDocs": {
+          "url": "https://msdn.microsoft.com/en-us/library/azure/mt639497.aspx"
```

Comment:
make sure this points to the right version. currently its pointing to older version of the spec.